prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -123,4 +137,96 @@ class LivdTableDownload() : CliktCommand(
             }
         return allLinks.filter { it.contains(partialHref) }
     }
+
+    /**
+     * Extracts LIVD table from sheet [sheetName] of the input Excel formate [inputfile] file, converts to
+     * csv format, and output to the CSV format [outpufile] file.
+     * @param sheetName is the LOINC Mapping sheet from the downloaded LOINC data code file.
+     * @param inputfile is the input Excel file name.
+     * @param inputfile is the output CSV file name.
+     * @return true for success and fales for failure.
+     */
+    private fun extractsLivdTable(sheetName: String, inputfile: String, outputfile: String): Boolean {
+
+        try {
+            val data = StringBuffer() // Buffer and output file for CSV data
+            val fileInputStream = FileInputStream(File(inputfile))
+            var workbook: Workbook? = null
+            val ext: String = FilenameUtils.getExtension(inputfile)
+            if (ext.equals("xlsx", ignoreCase = true)) {
+                workbook = XSSFWorkbook(fileInputStream)
+            } else if (ext.equals("xls", ignoreCase = true)) {
+                workbook = HSSFWorkbook(fileInputStream)
+            }
+
+            // Check for output file duplication
+            if (File(outputfile).exists()) {
+                val c = TermUi.prompt("\t$outputfile file is already existed: You want to overwrite it (y/n)?", "n")
+                if (c?.lowercase() == "n") {
+                    return false
+                } else {
+                    TermUi.echo("\tThe $outputfile file is overwriting.")
+                }
+            }
+
+            val fileOutputStream = FileOutputStream(File(outputfile))
+
+            // Get the LOINC Mapping sheet
+            val sheet: Sheet? = workbook!!.getSheet(sheetName)
+            if (sheet == null) {
+                print("\tERROR: Sheet \"$sheetName\" doesn't exist in the $inputfile file.")
```

Comment:
You are mixing different types of output.  If you just want to print then use TermUI like in other lines.

Reply:
Done.